tutor/commands/compose.py

```diff
@@ -187,12 +200,24 @@ def interactive_upgrade(
 
 
 def interactive_configuration(
-    context: click.Context, interactive: bool, run_for_prod: t.Optional[bool] = None
+    context: click.Context,
+    interactive: bool,
+    run_for_prod: t.Optional[bool] = None,
+    clean_environment: bool = False,
 ) -> None:
-    click.echo(fmt.title("Interactive platform configuration"))
     config = tutor_config.load_minimal(context.obj.root)
     if interactive:
-        interactive_config.ask_questions(config, run_for_prod=run_for_prod)
+        click.echo(fmt.title("Interactive platform configuration"))
+        interactive_config.ask_questions(
+            config,
+            context.obj.root,
+            run_for_prod=run_for_prod,
+            clean_environment=clean_environment,
+        )
+    else:
```

Comment:
I don't understand why this is needed here. Environment deletion will be handled by the save command, right?

Reply:
To have a launch -- clean command

Reply:
Yes, but this will add the clean flag to launch too

Reply:
launch flag removed.